modules/gapi/include/opencv2/gapi/gproto.hpp

```diff
@@ -135,7 +135,7 @@ GRunArg value_of(const GOrigin &origin);
 // Transform run-time computation arguments into a collection of metadata
 // extracted from that arguments
 GMetaArg  GAPI_EXPORTS descr_of(const GRunArg  &arg );
-GMetaArgs GAPI_EXPORTS_W descr_of(const GRunArgs &args);
+GMetaArgs GAPI_EXPORTS descr_of(const GRunArgs &args);
```

Comment:
This was used for streaming, but it is possible to compile streaming without meta, lets' do it later. This will be implemented via callback as well

Reply:
>  but it is possible to compile streaming without meta

Not always, unfortunately.

Reply:
Ok, then, I'll also add callback for that

Reply:
Added